target/linux/ramips/base-files/lib/ramips.sh

```diff
@@ -559,6 +559,9 @@ ramips_board_detect() {
 	*"WCR-150GN")
 		name="wcr-150gn"
 		;;
+        *"WD-03")
```

Comment:
Drop the WD-03 from this file and use the board compatible string from the dts as userspace boardname, as all recently added boards are do.